### PR TITLE
Support imagine runtime config in display url templating helper

### PR DESCRIPTION
**Pull request by aramalipoor**
| Q             | A
| ------------- | ---
| Doc fix?      | no
| New docs?     | yes symfony-cmf/MediaBundle#143
| Applies to    | MediaBundle
| Fixed tickets | -

### CODE REVIEW

File: bundles/media/introduction.rst [hunk 1]
```diff
@@ -216,6 +216,19 @@ The media bundle contains a Twig extension, it contains the following functions:
...
+    
+      .. code-block:: html+jinja
+
+        <a href="{{ cmf_media_display_url(file, {
+           imagine_filter: 'my_filter',
+           imagine_runtime_config: {
+              'thumbnail': {'size': [120, 60]}
+           }
+        }) }}" title="Download">Download</a>
+      .. versionadded:: 1.3
```
Comment (dbu) [inline bundles/media/introduction.rst]: this needs to be not indented. please add a blank line after the code and then make this line without whitespace in front.

File: bundles/media/introduction.rst [hunk 2]
```diff
@@ -216,6 +216,19 @@ The media bundle contains a Twig extension, it contains the following functions:
...
+        Option `imagine_runtime_config` was instroduced in MediaBundle 1.3
```
Comment (dbu) [inline bundles/media/introduction.rst]: s/instroduced/introduced

**Comment by aramalipoor** [comment]
@dbu Thanks for the review, anything else :) ?

**Comment by dbu** [comment]
thanks!

**Pull request merged**